src/presenter/event-presenter.js

```diff
@@ -1,35 +1,90 @@
 import { render } from '../render';
-import EventEditView from '../view/main/event-edit-view';
-import EventItemView from '../view/main/event-item-view';
-import EventListView from '../view/main/event-list-view';
-import EventView from '../view/main/event-view';
-import MainView from '../view/main/main-view';
-import SortView from '../view/main/sort-view';
-import TripView from '../view/main/trip-view';
+import EventEditView from '../view/event-edit-view';
+import EventItemView from '../view/event-item-view';
+import EventView from '../view/event-view';
+import EventsListView from '../view/events-list-view';
+import SortView from '../view/sort-view';
 
 export default class EventPresenter {
-  mainComponent = new MainView();
-  tripComponent = new TripView();
   sortComponent = new SortView();
-  eventListComponet = new EventListView();
-  editComponent = new EventEditView();
+  eventListComponent = new EventsListView();
 
-  constructor ({mainContainer}){
-    this.mainContainer = mainContainer;
+  constructor({ container, eventsModel }) {
+    this.container = container;
+    this.eventsModel = eventsModel;
   }
 
-  init() {
-    render(this.mainComponent, this.mainContainer);
-    render(this.tripComponent, this.mainComponent.getElement());
+  renderEventItem(callback) {
```

Comment:
Лучше использовать семантичные названия параметров

```suggestion
  renderEventItem(renderCallback) {
```